atlp-rwanda/commanders-rn-medica
# Revert "Feature: Sign Up and Sign In (#34)"

#### What does this PR do?
This PR reverts the changes introduced in the Feature: Sign Up and Sign In (#34) to restore the previous state.

#### Description of Task to be completed?
- Reverted the SignUp screen with email and password input fields.
- Reverted the SignIn screen with email and password input fields.
- Removed the active state implementation for the input fields.

#### How should this be manually tested?
1. Run the app.
2. Verify that the SignUp and SignIn screens have been reverted to their previous states.
3. Ensure that no new active state features are present in the input fields.

#### Any background context you want to provide?
The original changes were merged prematurely. This revert ensures that further review and approval can take place before merging again.

#### Questions:
Any questions or concerns about the revert process?
